frontend/assets/css/main.css

```
@@ -221,14 +221,13 @@
 
-/* Notifications: larger size */
-.vue-notification-wrapper {
-  width: 520px !important;
-}
-
+/* Notifications: larger text, wrapping */
 .vue-notification {
   font-size: 17px !important;
   padding: 18px 24px !important;
+  white-space: normal !important;
+  word-break: break-word !important;
```

Stylelint (17.11.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)